.cirrus.yml

```diff
@@ -21,8 +21,11 @@ env:
 
 config_template: &CONFIG_TEMPLATE
   auto_cancellation: $CIRRUS_BRANCH != 'master'
-  only_if: $CIRRUS_CRON != ''
-  skip: "changesIncludeOnly('doc/**', '**.{md,rdoc,ronn,[1-8]}', '.document')"
+  only_if: changesInclude('yjit.{c,h,rb}', 'yjit/**.{mk,rs,toml}')
```

Comment:
This seems irrelevant to `task`. I think this should be something like https://github.com/ruby/ruby/pull/8019.

If the `task` is there only for YJIT, I think we should delete `task` and resurrect a clang-12 job for `task_yjit` so that it can test bindgen with libclang.

Reply:
🤔 I changed my mind. I think this is okay as is.